### DISCUSSION TITLE
PG.MaxLFQ even with disabled cross run normalization

### DISCUSSION THREAD

**Discussion by serenacamerini73** [vdemichev/DiaNN#844] [Q&A]
I performed a data analysis on five samples: for this specific case I wouldn't normalize the data because I would appreciate the differences among  the volume injections. So, I disabled cross run normalization (OFF).

The results in the file report.tsv were:  PG.Quantity    PG.Normalised     PG.MaxLFQ   

The pg.matrix conained values corresponding to PG.Max.LFQ.

Should I consider that these are anyway normalized, even if I swiched off cross run normalization?
Thank you a lot

Serena

**Comment by vdemichev** [comment]
Hi Serena,

If you switch off cross run normalisation the PG.MaxLFQ column contains non-normalised data.

Best,
Vadim
